dydx3/starkex/conditional_transfer.py

```diff
@@ -45,14 +47,17 @@ def __init__(
             else int(receiver_public_key, 16)
         )
         quantums_amount = to_quantums_exact(human_amount, COLLATERAL_ASSET)
+        expiration_epoch_hours = math.ceil(
+            float(expiration_epoch_seconds) / ONE_HOUR_IN_SECONDS,
```

Comment:
do we want to leave `expiration_epoch_seconds` as the "public" value (the one the client sets)?

Reply:
Will do this separately